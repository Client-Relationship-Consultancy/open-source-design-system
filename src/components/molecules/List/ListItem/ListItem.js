import React from "react"
import styled from "styled-components"
import PropTypes from "prop-types"
import { colourPalette } from "../../../../brandColours.ts"

const StyledListItem = styled.li`
  padding: ${props => (props.padding ? "0.5rem 1rem" : "0")};
  border-bottom: 1px ${props => props.theme.black.tint20.hex} solid;
`
StyledListItem.defaultProps = {
  theme: colourPalette.examplePalette,
}
StyledListItem.displayName = "StyledListItem"

export const ListItem = props => {
  const { children, padding, id, className } = props
  return (
    <StyledListItem padding={padding} id={id} className={className}>
      {children}
    </StyledListItem>
  )
}
ListItem.displayName = "ListItem"
ListItem.defaultProps = {
  padding: true,
}
ListItem.propTypes = {
  padding: PropTypes.bool,
  children: PropTypes.node,
  id: PropTypes.string,
  className: PropTypes.string,
}

export default ListItem
