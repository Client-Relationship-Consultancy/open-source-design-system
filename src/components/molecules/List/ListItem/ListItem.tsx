import React from "react"
import styled from "styled-components"
import PropTypes from "prop-types"
import { colourPalette } from "../../../../brandColours"

interface IStyledListItem {
  padding?: boolean;
}

const StyledListItem = styled.li<IStyledListItem>`
  padding: ${props => (props.padding ? "0.5rem 1rem" : "0")};
  border-bottom: 1px ${props => props.theme.black.tint20.hex} solid;
`
StyledListItem.defaultProps = {
  theme: colourPalette.examplePalette,
}
StyledListItem.displayName = "StyledListItem"

type IListItem = React.HTMLProps<HTMLInputElement> & IStyledListItem

export const ListItem: React.FC<IListItem> = (props) => {
  const { children, padding, id, className } = props
  const styledListItemClassName = className
    ? `ListItem__StyledListItem- ${className}`
    : "ListItem__StyledListItem-"
  return (
    <StyledListItem padding={padding} id={id} className={styledListItemClassName}>
      {children}
    </StyledListItem>
  )
}
ListItem.displayName = "ListItem"
ListItem.defaultProps = {
  padding: true,
}

export default ListItem
