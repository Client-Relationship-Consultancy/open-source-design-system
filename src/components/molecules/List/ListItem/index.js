import ListItem, { IListItem } from "./ListItem"

export { IListItem }
export default ListItem
