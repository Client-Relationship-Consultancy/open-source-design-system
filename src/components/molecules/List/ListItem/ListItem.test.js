import React from "react"
import { mount } from "enzyme"
import ListItem from "./ListItem"
import "jest-styled-components"

describe("List Items Testing", () => {
  const text = "Example Item"
  const component = mount(<ListItem>{text}</ListItem>)
  it("Wraps items in <li> tags", () => {
    expect(component.find("li").text()).toEqual(text)
  })
  it("Default prop of padding is true", () => {
    expect(component.prop("padding")).toEqual(true)
    expect(component).not.toHaveStyleRule("padding", "0")
  })
  it("When padding is false, padding is 0", () => {
    component.setProps({ padding: false })
    expect(component).toHaveStyleRule("padding", "0")
  })
  it("Snapshot Testing", () => {
    expect(component).toMatchSnapshot()
  })

  it("Snapshot Testing with className", () => {
    const componentWithClassName = mount(<ListItem className="testClassName">{text}</ListItem>)

    expect(componentWithClassName).toMatchSnapshot()
  })
})
