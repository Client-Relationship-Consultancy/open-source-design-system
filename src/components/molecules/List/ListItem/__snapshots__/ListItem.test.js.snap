// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Items Testing Snapshot Testing 1`] = `
.c0 {
  padding: 0;
  border-bottom: 1px #ECECEC solid;
}

<ListItem
  padding={false}
>
  <StyledListItem
    className="ListItem__StyledListItem"
    padding={false}
    theme={
      Object {
        "action": Object {
          "dark": Object {
            "RGB": "79,38,98",
            "hex": "#4E2662",
            "on": "#FFFFFF",
          },
          "light": Object {
            "RGB": "173,101,207",
            "hex": "#AD65CF",
            "on": "#FFFFFF",
          },
          "main": Object {
            "RGB": "116,60,143",
            "hex": "#743C8F",
            "on": "#FFFFFF",
          },
        },
        "background": Object {
          "RGB": "246,246,246",
          "hex": "#F6F6F6",
        },
        "black": Object {
          "main": Object {
            "RGB": "39,39,39",
            "hex": "#272727",
            "on": "#FFFFFF",
          },
          "tint20": Object {
            "RGB": "236,236,236",
            "hex": "#ECECEC",
            "on": "#272727",
          },
          "tint40": Object {
            "RGB": "199,199,199",
            "hex": "#C7C7C7",
            "on": "#272727",
          },
          "tint60": Object {
            "RGB": "160,160,160",
            "hex": "#A0A0A0",
            "on": "#FFFFFF",
          },
          "tint80": Object {
            "RGB": "105,105,105",
            "hex": "#696969",
            "on": "#FFFFFF",
          },
        },
        "complimentary": Object {
          "dark": Object {
            "RGB": "206,185,0",
            "hex": "#CEB900",
            "on": "#272727",
          },
          "light": Object {
            "RGB": "255,245,157",
            "hex": "#FFF59D",
            "on": "#272727",
          },
          "main": Object {
            "RGB": "255,238,82",
            "hex": "#FFEE52",
            "on": "#272727",
          },
        },
        "error": Object {
          "dark": Object {
            "RGB": "197,73,1",
            "hex": "#C54901",
            "on": "#FFFFFF",
          },
          "main": Object {
            "RGB": "255,115,32",
            "hex": "#FF7320",
            "on": "#FFFFFF",
          },
        },
        "primary": Object {
          "dark": Object {
            "RGB": "102,22,20",
            "hex": "#661614",
            "on": "#FFFFFF",
          },
          "light": Object {
            "RGB": "218,52,47",
            "hex": "#DA342F",
            "on": "#FFFFFF",
          },
          "main": Object {
            "RGB": "164,40,36",
            "hex": "#A42824",
            "on": "#FFFFFF",
          },
        },
        "secondary": Object {
          "dark": Object {
            "RGB": "5,38,59",
            "hex": "#05263B",
            "on": "#FFFFFF",
          },
          "light": Object {
            "RGB": "89,156,175",
            "hex": "#599CAF",
            "on": "#FFFFFF",
          },
          "main": Object {
            "RGB": "11,70,107",
            "hex": "#0B466B",
            "on": "#FFFFFF",
          },
        },
        "surface": Object {
          "RGB": "255,255,255",
          "hex": "#FFFFFF",
          "on": "#272727",
        },
        "white": Object {
          "RGB": "255,255,255",
          "hex": "#FFFFFF",
          "on": "#272727",
        },
      }
    }
  >
    <StyledComponent
      className="ListItem__StyledListItem"
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "_foldedDefaultProps": Object {
            "theme": Object {
              "action": Object {
                "dark": Object {
                  "RGB": "79,38,98",
                  "hex": "#4E2662",
                  "on": "#FFFFFF",
                },
                "light": Object {
                  "RGB": "173,101,207",
                  "hex": "#AD65CF",
                  "on": "#FFFFFF",
                },
                "main": Object {
                  "RGB": "116,60,143",
                  "hex": "#743C8F",
                  "on": "#FFFFFF",
                },
              },
              "background": Object {
                "RGB": "246,246,246",
                "hex": "#F6F6F6",
              },
              "black": Object {
                "main": Object {
                  "RGB": "39,39,39",
                  "hex": "#272727",
                  "on": "#FFFFFF",
                },
                "tint20": Object {
                  "RGB": "236,236,236",
                  "hex": "#ECECEC",
                  "on": "#272727",
                },
                "tint40": Object {
                  "RGB": "199,199,199",
                  "hex": "#C7C7C7",
                  "on": "#272727",
                },
                "tint60": Object {
                  "RGB": "160,160,160",
                  "hex": "#A0A0A0",
                  "on": "#FFFFFF",
                },
                "tint80": Object {
                  "RGB": "105,105,105",
                  "hex": "#696969",
                  "on": "#FFFFFF",
                },
              },
              "complimentary": Object {
                "dark": Object {
                  "RGB": "206,185,0",
                  "hex": "#CEB900",
                  "on": "#272727",
                },
                "light": Object {
                  "RGB": "255,245,157",
                  "hex": "#FFF59D",
                  "on": "#272727",
                },
                "main": Object {
                  "RGB": "255,238,82",
                  "hex": "#FFEE52",
                  "on": "#272727",
                },
              },
              "error": Object {
                "dark": Object {
                  "RGB": "197,73,1",
                  "hex": "#C54901",
                  "on": "#FFFFFF",
                },
                "main": Object {
                  "RGB": "255,115,32",
                  "hex": "#FF7320",
                  "on": "#FFFFFF",
                },
              },
              "primary": Object {
                "dark": Object {
                  "RGB": "102,22,20",
                  "hex": "#661614",
                  "on": "#FFFFFF",
                },
                "light": Object {
                  "RGB": "218,52,47",
                  "hex": "#DA342F",
                  "on": "#FFFFFF",
                },
                "main": Object {
                  "RGB": "164,40,36",
                  "hex": "#A42824",
                  "on": "#FFFFFF",
                },
              },
              "secondary": Object {
                "dark": Object {
                  "RGB": "5,38,59",
                  "hex": "#05263B",
                  "on": "#FFFFFF",
                },
                "light": Object {
                  "RGB": "89,156,175",
                  "hex": "#599CAF",
                  "on": "#FFFFFF",
                },
                "main": Object {
                  "RGB": "11,70,107",
                  "hex": "#0B466B",
                  "on": "#FFFFFF",
                },
              },
              "surface": Object {
                "RGB": "255,255,255",
                "hex": "#FFFFFF",
                "on": "#272727",
              },
              "white": Object {
                "RGB": "255,255,255",
                "hex": "#FFFFFF",
                "on": "#272727",
              },
            },
          },
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "ListItem__StyledListItem-sc-1c1fhgc-0",
            "isStatic": false,
            "lastClassName": "c0",
            "rules": Array [
              "padding:",
              [Function],
              ";border-bottom:1px ",
              [Function],
              " solid;",
            ],
          },
          "displayName": "StyledListItem",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "ListItem__StyledListItem-sc-1c1fhgc-0",
          "target": "li",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
      padding={false}
      theme={
        Object {
          "action": Object {
            "dark": Object {
              "RGB": "79,38,98",
              "hex": "#4E2662",
              "on": "#FFFFFF",
            },
            "light": Object {
              "RGB": "173,101,207",
              "hex": "#AD65CF",
              "on": "#FFFFFF",
            },
            "main": Object {
              "RGB": "116,60,143",
              "hex": "#743C8F",
              "on": "#FFFFFF",
            },
          },
          "background": Object {
            "RGB": "246,246,246",
            "hex": "#F6F6F6",
          },
          "black": Object {
            "main": Object {
              "RGB": "39,39,39",
              "hex": "#272727",
              "on": "#FFFFFF",
            },
            "tint20": Object {
              "RGB": "236,236,236",
              "hex": "#ECECEC",
              "on": "#272727",
            },
            "tint40": Object {
              "RGB": "199,199,199",
              "hex": "#C7C7C7",
              "on": "#272727",
            },
            "tint60": Object {
              "RGB": "160,160,160",
              "hex": "#A0A0A0",
              "on": "#FFFFFF",
            },
            "tint80": Object {
              "RGB": "105,105,105",
              "hex": "#696969",
              "on": "#FFFFFF",
            },
          },
          "complimentary": Object {
            "dark": Object {
              "RGB": "206,185,0",
              "hex": "#CEB900",
              "on": "#272727",
            },
            "light": Object {
              "RGB": "255,245,157",
              "hex": "#FFF59D",
              "on": "#272727",
            },
            "main": Object {
              "RGB": "255,238,82",
              "hex": "#FFEE52",
              "on": "#272727",
            },
          },
          "error": Object {
            "dark": Object {
              "RGB": "197,73,1",
              "hex": "#C54901",
              "on": "#FFFFFF",
            },
            "main": Object {
              "RGB": "255,115,32",
              "hex": "#FF7320",
              "on": "#FFFFFF",
            },
          },
          "primary": Object {
            "dark": Object {
              "RGB": "102,22,20",
              "hex": "#661614",
              "on": "#FFFFFF",
            },
            "light": Object {
              "RGB": "218,52,47",
              "hex": "#DA342F",
              "on": "#FFFFFF",
            },
            "main": Object {
              "RGB": "164,40,36",
              "hex": "#A42824",
              "on": "#FFFFFF",
            },
          },
          "secondary": Object {
            "dark": Object {
              "RGB": "5,38,59",
              "hex": "#05263B",
              "on": "#FFFFFF",
            },
            "light": Object {
              "RGB": "89,156,175",
              "hex": "#599CAF",
              "on": "#FFFFFF",
            },
            "main": Object {
              "RGB": "11,70,107",
              "hex": "#0B466B",
              "on": "#FFFFFF",
            },
          },
          "surface": Object {
            "RGB": "255,255,255",
            "hex": "#FFFFFF",
            "on": "#272727",
          },
          "white": Object {
            "RGB": "255,255,255",
            "hex": "#FFFFFF",
            "on": "#272727",
          },
        }
      }
    >
      <li
        className="c0 ListItem__StyledListItem"
      >
        Example Item
      </li>
    </StyledComponent>
  </StyledListItem>
</ListItem>
`;

exports[`List Items Testing Snapshot Testing with className 1`] = `
.c0 {
  padding: 0.5rem 1rem;
  border-bottom: 1px #ECECEC solid;
}

<ListItem
  className="testClassName"
  padding={true}
>
  <StyledListItem
    className="ListItem__StyledListItem testClassName"
    padding={true}
    theme={
      Object {
        "action": Object {
          "dark": Object {
            "RGB": "79,38,98",
            "hex": "#4E2662",
            "on": "#FFFFFF",
          },
          "light": Object {
            "RGB": "173,101,207",
            "hex": "#AD65CF",
            "on": "#FFFFFF",
          },
          "main": Object {
            "RGB": "116,60,143",
            "hex": "#743C8F",
            "on": "#FFFFFF",
          },
        },
        "background": Object {
          "RGB": "246,246,246",
          "hex": "#F6F6F6",
        },
        "black": Object {
          "main": Object {
            "RGB": "39,39,39",
            "hex": "#272727",
            "on": "#FFFFFF",
          },
          "tint20": Object {
            "RGB": "236,236,236",
            "hex": "#ECECEC",
            "on": "#272727",
          },
          "tint40": Object {
            "RGB": "199,199,199",
            "hex": "#C7C7C7",
            "on": "#272727",
          },
          "tint60": Object {
            "RGB": "160,160,160",
            "hex": "#A0A0A0",
            "on": "#FFFFFF",
          },
          "tint80": Object {
            "RGB": "105,105,105",
            "hex": "#696969",
            "on": "#FFFFFF",
          },
        },
        "complimentary": Object {
          "dark": Object {
            "RGB": "206,185,0",
            "hex": "#CEB900",
            "on": "#272727",
          },
          "light": Object {
            "RGB": "255,245,157",
            "hex": "#FFF59D",
            "on": "#272727",
          },
          "main": Object {
            "RGB": "255,238,82",
            "hex": "#FFEE52",
            "on": "#272727",
          },
        },
        "error": Object {
          "dark": Object {
            "RGB": "197,73,1",
            "hex": "#C54901",
            "on": "#FFFFFF",
          },
          "main": Object {
            "RGB": "255,115,32",
            "hex": "#FF7320",
            "on": "#FFFFFF",
          },
        },
        "primary": Object {
          "dark": Object {
            "RGB": "102,22,20",
            "hex": "#661614",
            "on": "#FFFFFF",
          },
          "light": Object {
            "RGB": "218,52,47",
            "hex": "#DA342F",
            "on": "#FFFFFF",
          },
          "main": Object {
            "RGB": "164,40,36",
            "hex": "#A42824",
            "on": "#FFFFFF",
          },
        },
        "secondary": Object {
          "dark": Object {
            "RGB": "5,38,59",
            "hex": "#05263B",
            "on": "#FFFFFF",
          },
          "light": Object {
            "RGB": "89,156,175",
            "hex": "#599CAF",
            "on": "#FFFFFF",
          },
          "main": Object {
            "RGB": "11,70,107",
            "hex": "#0B466B",
            "on": "#FFFFFF",
          },
        },
        "surface": Object {
          "RGB": "255,255,255",
          "hex": "#FFFFFF",
          "on": "#272727",
        },
        "white": Object {
          "RGB": "255,255,255",
          "hex": "#FFFFFF",
          "on": "#272727",
        },
      }
    }
  >
    <StyledComponent
      className="ListItem__StyledListItem testClassName"
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "_foldedDefaultProps": Object {
            "theme": Object {
              "action": Object {
                "dark": Object {
                  "RGB": "79,38,98",
                  "hex": "#4E2662",
                  "on": "#FFFFFF",
                },
                "light": Object {
                  "RGB": "173,101,207",
                  "hex": "#AD65CF",
                  "on": "#FFFFFF",
                },
                "main": Object {
                  "RGB": "116,60,143",
                  "hex": "#743C8F",
                  "on": "#FFFFFF",
                },
              },
              "background": Object {
                "RGB": "246,246,246",
                "hex": "#F6F6F6",
              },
              "black": Object {
                "main": Object {
                  "RGB": "39,39,39",
                  "hex": "#272727",
                  "on": "#FFFFFF",
                },
                "tint20": Object {
                  "RGB": "236,236,236",
                  "hex": "#ECECEC",
                  "on": "#272727",
                },
                "tint40": Object {
                  "RGB": "199,199,199",
                  "hex": "#C7C7C7",
                  "on": "#272727",
                },
                "tint60": Object {
                  "RGB": "160,160,160",
                  "hex": "#A0A0A0",
                  "on": "#FFFFFF",
                },
                "tint80": Object {
                  "RGB": "105,105,105",
                  "hex": "#696969",
                  "on": "#FFFFFF",
                },
              },
              "complimentary": Object {
                "dark": Object {
                  "RGB": "206,185,0",
                  "hex": "#CEB900",
                  "on": "#272727",
                },
                "light": Object {
                  "RGB": "255,245,157",
                  "hex": "#FFF59D",
                  "on": "#272727",
                },
                "main": Object {
                  "RGB": "255,238,82",
                  "hex": "#FFEE52",
                  "on": "#272727",
                },
              },
              "error": Object {
                "dark": Object {
                  "RGB": "197,73,1",
                  "hex": "#C54901",
                  "on": "#FFFFFF",
                },
                "main": Object {
                  "RGB": "255,115,32",
                  "hex": "#FF7320",
                  "on": "#FFFFFF",
                },
              },
              "primary": Object {
                "dark": Object {
                  "RGB": "102,22,20",
                  "hex": "#661614",
                  "on": "#FFFFFF",
                },
                "light": Object {
                  "RGB": "218,52,47",
                  "hex": "#DA342F",
                  "on": "#FFFFFF",
                },
                "main": Object {
                  "RGB": "164,40,36",
                  "hex": "#A42824",
                  "on": "#FFFFFF",
                },
              },
              "secondary": Object {
                "dark": Object {
                  "RGB": "5,38,59",
                  "hex": "#05263B",
                  "on": "#FFFFFF",
                },
                "light": Object {
                  "RGB": "89,156,175",
                  "hex": "#599CAF",
                  "on": "#FFFFFF",
                },
                "main": Object {
                  "RGB": "11,70,107",
                  "hex": "#0B466B",
                  "on": "#FFFFFF",
                },
              },
              "surface": Object {
                "RGB": "255,255,255",
                "hex": "#FFFFFF",
                "on": "#272727",
              },
              "white": Object {
                "RGB": "255,255,255",
                "hex": "#FFFFFF",
                "on": "#272727",
              },
            },
          },
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "ListItem__StyledListItem-sc-1c1fhgc-0",
            "isStatic": false,
            "lastClassName": "c0",
            "rules": Array [
              "padding:",
              [Function],
              ";border-bottom:1px ",
              [Function],
              " solid;",
            ],
          },
          "displayName": "StyledListItem",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "ListItem__StyledListItem-sc-1c1fhgc-0",
          "target": "li",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
      padding={true}
      theme={
        Object {
          "action": Object {
            "dark": Object {
              "RGB": "79,38,98",
              "hex": "#4E2662",
              "on": "#FFFFFF",
            },
            "light": Object {
              "RGB": "173,101,207",
              "hex": "#AD65CF",
              "on": "#FFFFFF",
            },
            "main": Object {
              "RGB": "116,60,143",
              "hex": "#743C8F",
              "on": "#FFFFFF",
            },
          },
          "background": Object {
            "RGB": "246,246,246",
            "hex": "#F6F6F6",
          },
          "black": Object {
            "main": Object {
              "RGB": "39,39,39",
              "hex": "#272727",
              "on": "#FFFFFF",
            },
            "tint20": Object {
              "RGB": "236,236,236",
              "hex": "#ECECEC",
              "on": "#272727",
            },
            "tint40": Object {
              "RGB": "199,199,199",
              "hex": "#C7C7C7",
              "on": "#272727",
            },
            "tint60": Object {
              "RGB": "160,160,160",
              "hex": "#A0A0A0",
              "on": "#FFFFFF",
            },
            "tint80": Object {
              "RGB": "105,105,105",
              "hex": "#696969",
              "on": "#FFFFFF",
            },
          },
          "complimentary": Object {
            "dark": Object {
              "RGB": "206,185,0",
              "hex": "#CEB900",
              "on": "#272727",
            },
            "light": Object {
              "RGB": "255,245,157",
              "hex": "#FFF59D",
              "on": "#272727",
            },
            "main": Object {
              "RGB": "255,238,82",
              "hex": "#FFEE52",
              "on": "#272727",
            },
          },
          "error": Object {
            "dark": Object {
              "RGB": "197,73,1",
              "hex": "#C54901",
              "on": "#FFFFFF",
            },
            "main": Object {
              "RGB": "255,115,32",
              "hex": "#FF7320",
              "on": "#FFFFFF",
            },
          },
          "primary": Object {
            "dark": Object {
              "RGB": "102,22,20",
              "hex": "#661614",
              "on": "#FFFFFF",
            },
            "light": Object {
              "RGB": "218,52,47",
              "hex": "#DA342F",
              "on": "#FFFFFF",
            },
            "main": Object {
              "RGB": "164,40,36",
              "hex": "#A42824",
              "on": "#FFFFFF",
            },
          },
          "secondary": Object {
            "dark": Object {
              "RGB": "5,38,59",
              "hex": "#05263B",
              "on": "#FFFFFF",
            },
            "light": Object {
              "RGB": "89,156,175",
              "hex": "#599CAF",
              "on": "#FFFFFF",
            },
            "main": Object {
              "RGB": "11,70,107",
              "hex": "#0B466B",
              "on": "#FFFFFF",
            },
          },
          "surface": Object {
            "RGB": "255,255,255",
            "hex": "#FFFFFF",
            "on": "#272727",
          },
          "white": Object {
            "RGB": "255,255,255",
            "hex": "#FFFFFF",
            "on": "#272727",
          },
        }
      }
    >
      <li
        className="c0 ListItem__StyledListItem testClassName"
      >
        Example Item
      </li>
    </StyledComponent>
  </StyledListItem>
</ListItem>
`;
